releasenotes/notes/update-qeomresult-docstring-and-pending-deprecate-mqwv-matrix-std-setters-5bc2d84a941dca72.yaml

```diff
@@ -0,0 +1,11 @@
+---
+features:
+  - |
+    Completed the missing documentation for the new attributes of the class
+    :class:`~qiskit_nature.second_q.algorithms.excited_state_solvers.qeom.QEOMResult`.
+
```

Comment:
```suggestion
```
Since the previous changes were not released yet, this note is not necessary

Reply:
Done